src/Workspaces/CSharpTest/Formatting/FormattingTests.cs

```diff
@@ -9861,6 +9861,109 @@ void Goo(R r)
 }", changedOptionSet: changingOptions);
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.Formatting)]
+        [WorkItem(52413, "https://github.com/dotnet/roslyn/issues/52413")]
+        public async Task NewLinesForBraces_PropertyPatternClauses_Default()
+        {
+            await AssertFormatAsync(
+                @"
+class A
+{
+    public string Name { get; }
+
+    public bool IsFoo(A a)
+    {
+        return a is
+        {
+            Name: ""foo"",
+        };
+    }
+}",
+                @"
+class A
+{
+    public string Name { get; }
+
+    public bool IsFoo(A a)
+    {
+        return a is {
+            Name: ""foo"",
+        };
+    }
+}");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Formatting)]
+        [WorkItem(52413, "https://github.com/dotnet/roslyn/issues/52413")]
+        public async Task NewLinesForBraces_PropertyPatternClauses_NonDefault()
+        {
+            var changingOptions = new OptionsCollection(LanguageNames.CSharp)
+            {
+                { NewLinesForBracesInObjectCollectionArrayInitializers, false },
+            };
+            await AssertFormatAsync(
+                @"
+class A
+{
+    public string Name { get; }
+
+    public bool IsFoo(A a)
+    {
+        return a is {
+            Name: ""foo"",
+        };
+    }
+}",
+                @"
+class A
+{
+    public string Name { get; }
+
+    public bool IsFoo(A a)
+    {
+        return a is
+        {
+            Name: ""foo"",
+        };
+    }
+}", changedOptionSet: changingOptions);
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Formatting)]
+        [WorkItem(52413, "https://github.com/dotnet/roslyn/issues/52413")]
+        public async Task NewLinesForBraces_PropertyPatternClauses_SingleLine()
+        {
+            var changingOptions = new OptionsCollection(LanguageNames.CSharp)
+            {
+                { NewLinesForBracesInObjectCollectionArrayInitializers, false },
+            };
+            await AssertFormatAsync(
+                @"
+class A
+{
+    public string Name { get; }
+
+    public bool IsFoo(A a)
+    {
+        return a is {
+            Name: ""foo"",
+        };
+    }
+}",
+                @"
+class A
+{
+    public string Name { get; }
+
+    public bool IsFoo(A a)
+    {
+        return a is {
+            Name: ""foo"",
+        };
+    }
+}", changedOptionSet: changingOptions);
+        }
```

Comment:
This test isn't actually testing the single-line form 😄 

```suggestion
        [Theory, CombinatorialData, Trait(Traits.Feature, Traits.Features.Formatting)]
        [WorkItem(52413, "https://github.com/dotnet/roslyn/issues/52413")]
        public async Task NewLinesForBraces_PropertyPatternClauses_SingleLine(bool option)
        {
            var changingOptions = new OptionsCollection(LanguageNames.CSharp)
            {
                { NewLinesForBracesInObjectCollectionArrayInitializers, option },
            };
            var code =
                @"
class A
{
    public string Name { get; }

    public bool IsFoo(A a)
    {
        return a is { Name: ""foo"" };
    }
}";

            await AssertFormatAsync(code, code, changedOptionSet: changingOptions);
        }
```

Reply:
I thought you meant something else completely by single line. Just saw the other SingleLine tests in this file. My bad, working on it.